psi4/src/psi4/dfocc/tei_grad_ref.cc

```diff
@@ -0,0 +1,335 @@
+/*
```

Comment:
This file is not used in `dfocc` as of #1953 and can be deleted.

Congratulations on passing tests.

Reply:
Thanks for the tip! It'll be gone in the next commit.